src/ghci/parse/module_set.rs

```diff
@@ -99,13 +111,32 @@ impl ModuleSet {
         }
     }
 
+    /// Format modules for adding or removing from a GHCi session.
+    ///
+    /// See [`ModuleSet::module_import_name`].
+    pub fn format_modules(
+        &self,
+        show_paths: &ShowPaths,
+        modules: &[NormalPath],
+    ) -> miette::Result<String> {
+        modules
+            .iter()
+            .map(|path| {
+                self.module_import_name(show_paths, path)
```

Comment:
These `ModuleSet` APIs are showing some strain, but it's not clear what to replace them with. I tried a few designs and didn't like the results.